examples/applications/plot_topics_extraction_with_nmf_lda.py

```diff
@@ -101,7 +103,14 @@ def plot_top_words(model, feature_names, n_top_words, title):
     "n_samples=%d and n_features=%d..." % (n_samples, n_features)
 )
 t0 = time()
-nmf = NMF(n_components=n_components, random_state=1, alpha=0.1, l1_ratio=0.5).fit(tfidf)
+nmf = NMF(
+    n_components=n_components,
+    random_state=1,
+    init=init,
+    alpha_W=0.00005,
+    alpha_H=0.00005,
+    l1_ratio=1,
+).fit(tfidf)
```

Comment:
For explicitness regarding the β-divergence.
```suggestion
nmf = NMF(
    n_components=n_components,
    random_state=1,
    init=init,
    beta_loss="frobenius",
    alpha_W=0.00005,
    alpha_H=0.00005,
    l1_ratio=1,
).fit(tfidf)
```

Also, since `l1_ratio` has been changed here, should a comment about the effect and/or choice of the L1 regularization be added here?

Or should both L1 and L2 regularization be used as previously?
```suggestion
nmf = NMF(
    n_components=n_components,
    random_state=1,
    init=init,
    alpha_W=0.00005,
    alpha_H=0.00005,
    l1_ratio=0.5,
).fit(tfidf)
```